runtime/src/iree-amd-aie/driver/xrt-lite/device.cc

```diff
@@ -202,6 +202,17 @@ static iree_status_t iree_hal_xrt_lite_device_queue_alloca(
   return iree_ok_status();
 }
 
+static iree_status_t iree_hal_xrt_lite_device_queue_dealloca(
```

Comment:
Heads up, this should be updated by adding another argument `iree_hal_alloca_flags_t flags` after the IREE bump merged. Ref https://github.com/nod-ai/iree-amd-aie/pull/1220/files#diff-33d7a4a056b0c413c9edc400cc4a454dc156c59783f6606c6b60e449a093f456R267